### DISCUSSION TITLE
Handling Partially Fulfilled Orders

### DISCUSSION THREAD

**Discussion by KrEnjuTSU** [saleor/saleor#11521] [General] [answered]
Hello everyone,

How to prevent partially fulfilled orders from blocking stocks for unfulfilled products? What is the proper way to release the stock reservation of the unfulfilled items of a partially fulfilled orders ? Cancelling the entire order does not make sense!

**Comment by karolkielecki** [comment, marked as answer]
Hi @KrEnjuTSU, click the gear icon on the right side of the order details page. Choose the return / replace order option. You can mark those unfulfilled items as returns. The products will be automatically restocked.
